settings.gradle.kts

```diff
@@ -131,359 +129,363 @@ include(":smoke-tests:images:servlet:servlet-3.0")
+hideFromDependabot(":instrumentation:spring:spring-rmi-4.0:javaagent")
+hideFromDependabot(":instrumentation:spring:spring-scheduling-3.1:javaagent")
+hideFromDependabot(":instrumentation:spring:spring-web-3.1:javaagent")
+hideFromDependabot(":instrumentation:spring:spring-web-3.1:library")
+hideFromDependabot(":instrumentation:spring:spring-web-3.1:testing")
+hideFromDependabot(":instrumentation:spring:spring-webmvc-3.1:javaagent")
+hideFromDependabot(":instrumentation:spring:spring-webmvc-3.1:wildfly-testing")
+hideFromDependabot(":instrumentation:spring:spring-webmvc-5.3:library")
+hideFromDependabot(":instrumentation:spring:spring-webflux-5.0:javaagent")
+hideFromDependabot(":instrumentation:spring:spring-webflux-5.0:library")
+hideFromDependabot(":instrumentation:spring:spring-ws-2.0:javaagent")
+hideFromDependabot(":instrumentation:spring:spring-boot-autoconfigure")
+hideFromDependabot(":instrumentation:spring:starters:spring-boot-starter")
+hideFromDependabot(":instrumentation:spring:starters:jaeger-spring-boot-starter")
+hideFromDependabot(":instrumentation:spring:starters:zipkin-spring-boot-starter")
+hideFromDependabot(":instrumentation:spymemcached-2.12:javaagent")
+hideFromDependabot(":instrumentation:struts-2.3:javaagent")
+hideFromDependabot(":instrumentation:tapestry-5.4:javaagent")
+hideFromDependabot(":instrumentation:tomcat:tomcat-7.0:javaagent")
+hideFromDependabot(":instrumentation:tomcat:tomcat-10.0:javaagent")
+hideFromDependabot(":instrumentation:tomcat:tomcat-common:javaagent")
+hideFromDependabot(":instrumentation:tomcat:tomcat-jdbc")
+hideFromDependabot(":instrumentation:twilio-6.6:javaagent")
+hideFromDependabot(":instrumentation:undertow-1.4:bootstrap")
+hideFromDependabot(":instrumentation:undertow-1.4:javaagent")
+hideFromDependabot(":instrumentation:vaadin-14.2:javaagent")
+hideFromDependabot(":instrumentation:vaadin-14.2:testing")
+hideFromDependabot(":instrumentation:vertx:vertx-http-client:vertx-http-client-3.0:javaagent")
+hideFromDependabot(":instrumentation:vertx:vertx-http-client:vertx-http-client-4.0:javaagent")
+hideFromDependabot(":instrumentation:vertx:vertx-http-client:vertx-http-client-common:javaagent")
+hideFromDependabot(":instrumentation:vertx:vertx-kafka-client-3.6:javaagent")
+hideFromDependabot(":instrumentation:vertx:vertx-kafka-client-3.6:testing")
+hideFromDependabot(":instrumentation:vertx:vertx-rx-java-3.5:javaagent")
+hideFromDependabot(":instrumentation:vertx:vertx-web-3.0:javaagent")
+hideFromDependabot(":instrumentation:vertx:vertx-web-3.0:testing")
+hideFromDependabot(":instrumentation:vibur-dbcp-11.0:javaagent")
+hideFromDependabot(":instrumentation:vibur-dbcp-11.0:library")
+hideFromDependabot(":instrumentation:vibur-dbcp-11.0:testing")
+hideFromDependabot(":instrumentation:wicket-8.0:javaagent")
 
 // benchmark
 include(":benchmark-overhead-jmh")
 include(":benchmark-jfr-analyzer")
+
+fun hideFromDependabot(projectPath: String) {
```

Comment:
```suggestion
// this effectively hides the submodule from dependabot because dependabot only regex parses gradle
// files looking for certain patterns
fun hideFromDependabot(projectPath: String) {
```